lib/validators/approvals.js

```diff
@@ -20,6 +21,7 @@ class Approvals extends Validator {
       'pull_request.unlabeled',
       'pull_request.synchronize'
     ]
+    this.supportedEvents.push('block')
```

Comment:
Should this be `pull_request.block` ? Thought I couldn't find such an event in the API docs...

Reply:
this should not be there, it was my mistake